elasticsearch/includes/classes/commands/class-healthcommand.php

```diff
@@ -80,7 +84,7 @@ private function render_results( array $results ) {
 		foreach( $results as $result ) {
 			// If it's an error, print out a warning and go to the next iteration
 			if ( array_key_exists( 'error', $result ) ) {
-				WP_CLI::warning( 'Error while validating count:' . $result->get_error_message() );
+				WP_CLI::warning( 'Error while validating count: ' . $result[ 'error' ]);
```

Comment:
With the Health helper functions returning WP_Error's now, does this need to be reworked to check `is_wp_error()`?

Reply:
Hmmm... No, because `validate_index_posts_count()` does not return `WP_Error`.
And in the case of `validate_users_count()`, I am checking the return value _before_ calling `render_results()`.

Reply:
Gotcha, LGTM then.